test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -125,7 +129,11 @@ Describe 'ConvertFrom-Markdown tests' -Tags 'CI' {
         BeforeAll {
             $mdFile = New-Item -Path $TestDrive/input.md -Value "Some **test string** to write in a file" -Force
             $mdLiteralPath = New-Item -Path $TestDrive/LiteralPath.md -Value "Some **test string** to write in a file" -Force
-            $expectedStringFromFile = "Some $esc[1mtest string$esc[0m to write in a file`n`n"
+            $expectedStringFromFile = if ($host.UI.SupportsVirtualTerminal) {
```

Comment:
should `$host.UI.SupportsVirtualTerminal` be `$VT100Support`?

Reply:
Moved the variable to Describe BeforeAll